testv2/testv2_auth/test_async_oauth.py

```
@@ -0,0 +1,369 @@
+"""
+Tests for the AsyncOAuth client that verifies it provides async versions of all OAuth operations.
+"""
+
+import pytest
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)